dm/worker/server.go

```diff
@@ -98,22 +98,22 @@ func (s *Server) Start() error {
 		// worker keepalive with master
 		// If worker loses connect from master, it would stop all task and try to connect master again.
 		shouldExit := false
+		var err1 error
 		for !shouldExit {
-			shouldExit, err = s.KeepAlive()
-			if err != nil || !shouldExit {
+			shouldExit, err1 = s.KeepAlive()
+			if err1 != nil || !shouldExit {
```

Comment:
how about print the err in log

Reply:
added in https://github.com/pingcap/dm/pull/453/commits/5b4177d2fda1da6edb01eef589000a7738d30087